server/service/src/sync/program_schemas/patient.json

```diff
@@ -375,10 +360,8 @@
           "$ref": "#/definitions/Allergies"
         },
         "birthOrder": {
-          "description": "Birth order, e.g. \"02\" for second child",
-          "examples": ["01", "02", "03"],
-          "pattern": "^[0-9]{2}$",
-          "type": "string"
+          "description": "Birth order, e.g. second child = 2",
+          "type": "number"
```

Comment:
Could you please add "minimum": 0 here? (@minimum 0 in the TS docs comment)

And maybe also "maximum": 99 to enforce two digits here... (the number control doesn't check for maximum at the moment though and could be added along the way)

Reply:
Cool: dae9eb1ec (both requests)